java8-browser-awscli-library/Dockerfile

```
@@ -0,0 +1,33 @@
+FROM openjdk:8-jdk-slim
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+      git \
+      openssh-client \
+      procps \
+      ca-certificates \
+      curl \
+      unzip \
+      gnupg
```

もし、aptを使って、2回に分けてパッケージをインストールするのであれば
その都度 `apt-get clean` するべきです

```suggestion
RUN apt-get update \
 && apt-get install -y --no-install-recommends \
      git \
      openssh-client \
      procps \
      ca-certificates \
      curl \
      unzip \
      gnupg \
 && apt-get clean && rm -rf /var/lib/apt/lists/*
```